app/assets/stylesheets/heliotrope.scss

```diff
@@ -300,6 +300,23 @@ footer.press {
     width: 100% !important;
   }
 
+  // TODO: WEBKIT MEDIA CONTROLS
+  // This should work for all webkit browsers
+  ::-internal-media-controls-download-button {
+     display:none;
+   }
+  // but doesn't in Chrome Version 55
+  // so we hide the download button (which is the last button on the panel)
+  ::-webkit-media-controls-panel {
+    width: calc(100% + 30px); /* Adjust as needed */
+  }
+  // by hiding the overflow of the enclosure
+  ::-webkit-media-controls-enclosure {
+    overflow:hidden;
+  }
+  // but this won't work for non-shadow-DOM browsers
+  // so this is a temporary fix to buy time.
+
```

Comment:
@gkostin1966 because this also impacts Safari, you could do a media query for Safari 7+ (that will also catch Chrome 22-28, but that doesn't matter b/c there isn't a DL button in Chrome 28). Try this at line 319:

```
@media \\0 screen {
  ::-webkit-media-controls-panel {
    width: 100% !important;
  }
}
```